app/src/state/ducks/auth.js

```diff
@@ -153,8 +153,11 @@ export function logout() {
 // Side Effects, e.g. thunks
 export function me() {
   return async (dispatch, getState, { api }) => {
-    if (!document.cookie.includes('token') && !process.env.TOKEN) {
-      // dont attempt if there is no token.
+    if (
+      process.env.NODE_ENV !== 'development' &&
```

Comment:
not quite sure why we need this line...

``` process.env.NODE_ENV !== 'development'```

Reply:
The cookie is being set by fetch based on the location of the API, on the api domain *.openelectionsportland.org as opposed to localhost. The client has no access to the token cookie except via fetch when connecting to the API domain.

Reply:
So the condition will always be true, on when the client domain is the not the same or a subdomain of the API.

Reply:
referencing in process.env.NODE_ENV is done in other parts of the code and should not be a problem in production.